packages/firestore/src/local/local_store.ts

```diff
@@ -163,6 +163,10 @@ export class LocalStore {
      */
     private garbageCollector: GarbageCollector
   ) {
+    assert(
+      persistence.started,
+      'Local Store passed an unstarted persistence implementation'
```

Comment:
The LocalStore is the receiver here, which doesn't quite come across in this message (maybe it is just missing a `was` as in `was passed `)

Reply:
Done.